tripy/tripy/frontend/module/module.py

```diff
@@ -203,7 +210,19 @@ def find_module(module: Union[Module, List, Dict], sub_strs: List[str]):
                     module = operator.attrgetter(child_name)(module)
             return module
 
+        expected_keys = set(self.state_dict())
+        provided_keys = set(state_dict)
+        missing_keys = expected_keys - provided_keys
+        unexpected_keys = provided_keys - expected_keys
+        if strict and (missing_keys or unexpected_keys):
+            raise_error(
+                "state_dict is incompatible.",
+                f"Got missing keys=[{', '.join(missing_keys)}], unexpected_keys=[{', '.join(unexpected_keys)}].",
```

Comment:
We can make the error a little bit nicer. Also we don't need the `join` since sets will be printed that way anyway:
```suggestion
            details = []
            if missing_keys:
                details.append(f"Missing keys: {missing_keys}\n")
            if unexpected_keys:
                details.append(f"Unexpected keys: {unexpected_keys}")
            raise_error(
                "state_dict is incompatible.",
                details,
```